lib/matplotlib/contour.py

```diff
@@ -575,6 +575,7 @@ def add_label_near(self, x, y, inline=True, inline_spacing=5,
         # Get label width for rotating labels and breaking contours
         lw = self.get_label_width(self.labelLevelList[lmin],
                                   self.labelFmt, self.labelFontSizeList[lmin])
+        lw *= self.ax.figure.dpi / 72.0
```

Comment:
Can you add a comment about units?  

From the conversion we are getting pts out of the `get_label_width` and `calc_label_rot_and_inline` is expecting pixels (and the doc strings seem to agree).

Reply:
I just copied the same line in the vanilla label below.  I’ll readily admit I don’t understand why this stuff is all such a mess.

Reply:
Because implicit units are hard :smiling_imp:  and we used to default to 80dpi on the screen so subtle things like this are easy to miss.

Reply:
OK, I get it now.   Its a little funny this uses the heuristic of 0.6 * fontsize * len(str) to get the width.  I thought we had better ways to do that, but maybe we pre-draw first to do that.  Anyway, that can be for a future day...